scripts/utils/pip/torizon_templates_utils/tasks.py

```diff
@@ -894,18 +894,19 @@ def run_task(self, label: str) -> None:
         if self.__gitlab_ci:
             _cmd = self.__replace_docker_host(_cmd)
 
-        # inject env
+        # Set env var for the task, adding the envs present on the task
+        task_env = os.environ.copy()
```

Comment:
this one is a trick one, because ok, it solves this issue when we need to set it by a task, but and if we set the env that is used on this task, it will overwrite it anyway or will use the env that I set before run tasks.xsh?